.github/actions/check-pull-request-commits/commitlint.config.js

```diff
@@ -0,0 +1,16 @@
+// (C) 2022 GoodData Corporation
+
+const LINE_WIDTH = 72;
```

Comment:
I'm not sure if 72 characters per line is enough, thoughts?

Reply:
I think it is fine, personally I stick with what VSCode suggests:
50 for the title, 72 for body.